cmd/kyma/alpha/create/module/module.go

```diff
@@ -81,34 +83,14 @@ func (cmd *command) Run(args []string) error {
 
 	l := cli.NewLogger(cmd.opts.Verbose).Sugar()
 
+	defaultCRValidated := false
```

Comment:
This boolean is redundant and calling `validateDefaultCR()` in 2 places as well.
 You can just do it once as an individual step in line 141:
 ```
 if cmd.opts.ValidateDefaultCR || cmd.opts.RegistryURL != "" {
    validateStep := cmd.NewStep("Validating resources...")

    // do validation checks here

    validateStep.Success()
} 
```

Reply:
@clebs OK, let me think about it.